server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/OrganizationTester.java

```diff
@@ -136,6 +136,6 @@ private void verifyMembersGroupMembership(String userLogin, @Nullable Organizati
   }
 
   public OrganizationService service() {
-    return session.wsClient().organizations();
+    return session.wsClient().organizationsOld();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "organizationsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)